app/components/views/GetStartedPage/hooks.js

```diff
@@ -145,12 +145,13 @@ export const useGetStarted = () => {
         console.log("is At Start Wallet");
         const { selectedWallet } = context;
         const { passPhrase } = context;
+        const isWatchingOnly = selectedWallet.value.isWatchingOnly;
         const hasPassPhrase = !!passPhrase;
         onStartWallet(selectedWallet, hasPassPhrase)
           .then((discoverAccountsComplete) => {
             setSelectedWallet(selectedWallet);
             const { passPhrase } = context;
-            if (!discoverAccountsComplete && !passPhrase) {
+            if (!discoverAccountsComplete && !passPhrase && !isWatchingOnly) {
```

Comment:
I guess you also need to add isTrezor, because it still asks me for the password if I check trezor but not wathing only.

Reply:
Trezors are also considered watching only wallets, and I tested the PR with a physical trezor and wasn't asked. What exactly was the flow you used that made you hit it?

Reply:
Updated to account for trezor wallets.